### DISCUSSION TITLE
【氵】祝各位绳匠都有永不完结的暑假！

### DISCUSSION THREAD

**Discussion by Fintbo** [share121/inter-knot#3037] [其他]
[![凉宫](https://github.com/user-attachments/assets/7eb4e85e-8d72-4885-bce5-96fccb344be0 "看我干嘛?还不赶快去写作业!")](https://www.bilibili.com/bangumi/play/ss1057)
~~同时默哀已经开学的绳匠们1秒钟~~

**Comment by fashibuding1214** [comment]
好好好
